### ISSUE TITLE
Onboarding is not very clear

### ISSUE THREAD

**Issue by trosel** [oakes/ansiwave#11]
as a new user, it's not super clear when i'm "on the network" (or what that really means in this case).

i read in other places that you manually approve users. does that mean that when someone first logs on, they are basically sitting in a fork of your database and they are submitting a pull request to the main database when they submit changes? 

if that is the case, then the app doesn't really have a concept of being "online", right? 

maybe some kind of status indicator on the screen could help to provide some context to the user in terms of "where they are" / "what their status is", if that makes sense

**Comment by oakes** [comment]
Actually, nothing happens when you first login. There is no server-side session; logging in just means you generated a login key (or added an existing one) locally. But when you make a post, it will initially end up in a separate db called "limbo" which i check regularly. Once i add you, all of your posts will be moved into the main db and all future posts will go there automatically.

You can see if you're in limbo by clicking on "my page". If you're in limbo, it will say so at the top. This will take a few seconds while it queries for your user. If you haven't made any posts at all, i believe it will not say that message because technically you don't exist anywhere yet. But yeah, it would be good to make this status more obvious, i'll try to think more about that. BTW i don't see anything in limbo right now, so i'm guessing you haven't made a post yet?

**Comment by trosel** [comment]
Ok that makes sense! I guess I was approved a long time ago. I think maybe the UI is just a little confusing for me (using the web client). 

![image](https://user-images.githubusercontent.com/1299322/152652362-17ec85ca-2d18-499d-8e39-643194800722.png)

So these three boxes are "boards" like subreddits. And on My Page, I can create a "journal post" where my journal is a personal "board" / subreddit where only i post things.

Have you thought about making an SDK for this so other people can try their hand at making alternate front ends?

**Comment by oakes** [comment]
Yeah those are the subboards right now, and journal posts are like blog posts.

The technique i use for querying makes it very open for alternative frontends. Once things settle down i plan on extracting it so others can use it more easily. There isn't even technically a need for an API, because the raw SQLite database is public and can be queried directly. I'll write more about this later but anyway, it should make it really practical to write clients in any language you want.
